### DISCUSSION TITLE
Audio bitrate

### DISCUSSION THREAD

**Discussion by Lucky390163** [Shabinder/SpotiFlyer#2256] [General]
Audio bitrate converting quality is do not get exact and same at any time.

**Comment by Lucky390163** [comment]
It download wrong bitrate in sometime even setting them properly.

**Comment by Lucky390163** [comment]
I have selected 192kbps and it downloads to 128 kbps😑.Everytime

**Comment by cochraneserver** [comment]
How did you specify the bitrate? I have a premium account but having trouble finding info on configuring settings outside of what is in the GUI (Windows).

**Comment by Lucky390163** [comment]
I use the poweramp app.

**Comment by IAM-Official** [comment]
Does it require premium subscription to download 320Kbps Quality? It's locked at 128Kbps, I already changed the Download Quality settings but the download result is still the same.

**Comment by Lucky390163** [comment]
No, it doesn't.

**Reply by IAM-Official** [reply]
Is there any solution to fix this bitrate problem? Or that's the devs job so we have to wait for them?
